### DISCUSSION TITLE
添加Socket调试工具

### DISCUSSION THREAD

**Discussion by MrAccle** [he3-app/he3-feedback#186] [Ideas]
可以快速启动一个基于TCP的Client/Server，并可以试试查看与发送通信报文
基于UDP发送通信报文

**Comment by MrAccle** [comment]
![image](https://github.com/he3-app/he3-feedback/assets/25242380/4b6a8b5e-9c0e-4c78-8805-f9d5c599e19d)
参考功能

**Comment by lyzhang1999** [comment]
感谢您的建议，我已经创建了 issues：https://github.com/he3-app/he3-feedback/issues/187

此外如果您有兴趣开发这款工具，也可以参考我们的开发文档：https://docs.he3app.com/guide/
